ansible/upgrade_sonic.yml

```diff
@@ -4,11 +4,41 @@
 #   ansible-playbook upgrade_sonic.yml -i lab -l devicename -e "upgrade_type=sonic" -e "image_url='http://8.8.8.8/sonic-broadcom.bin'"
 # upgrade via onie:
 #   ansible-playbook upgrade_sonic.yml -i lab -l devicename -e "upgrade_type=onie" -e "image_url='http://8.8.8.8/sonic-broadcom.bin'"
-
+# upgrade DUTs listed in testbed
+#   ansible-playbook upgrade-sonic.yaml -i lab -e "testbed_name=vms1-1" -e "testbed_file=testbed.csv" -e "upgrade_type=onie" -e "image_url='http://8.8.8.8/sonic-broadcom.bin'"
 - hosts: all
   gather_facts: no
   tasks:
 
+    - name: Add DUTs defined in testbed
+      block:
+        - name: Set default testbed file
+          set_fact:
+            testbed_file: testbed.csv
```

Comment:
Can this job support testbed.yaml since we are moving towards using testbed.yaml?

Reply:
Yes, uses can pass `testbed.yaml` like the following to use it:
```
ansible-playbook upgrade-sonic.yaml -i lab -e "testbed_name=vms1-1" -e "testbed_file=testbed.csv" -e "upgrade_type=onie" -e "image_url='http://8.8.8.8/sonic-broadcom.bin'"
```
In the future, we could modify the default to `testbed.yaml` as well.